Things.md

```diff
@@ -0,0 +1,117 @@
+The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize` (i.e. `unitsize` of 1 means no scaling, `unitsize` of 2 means in-game measurements are doubled).  
+There are a number of ways to move Things, all of which are done by manipulating these 4 properties.  
+```javascript
+// Instantiates Things used in the following code samples.
+var thing = FSM.GroupHolder.getSolid(0);
+var otherThing = FSM.GroupHolder.getSolid(1);
+```
+
+#### Shift  
+You can use the shift functions `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
+```javascript
+// Shifts a Thing vertically by 1.
+FSM.shiftVert(thing, 1);
+
+// Shifts a Thing horizontally by 1. 
+FSM.shiftHoriz(thing, 1);
+```
+
+#### Set  
+The `set` function (`setTop`, `setRight`, `setBottom`, or `setLeft`) moves the Thing to line up with the bounding property.
+```javascript
+// Sets the top bound of a Thing to 10. 
+FSM.setTop(thing, 10);
+
+// Sets the left bound of a Thing to 10. 
+FSM.setLeft(thing, 10);
+
+// Sets the bottom bound of a Thing to 10. 
+FSM.setBottom(thing, 10);
+
+// Sets the right bound of a Thing to 10. 
+FSM.setRight(thing, 10);
+```  
+
+#### Midpoints  
+It's possible to set the Thing's midpoint using `setMidX`, `setMidY`, and `setMid` (the latter of which sets both the `x` and `y` midpoints). These functions make it so the Thing is centered on the given `x` and `y`.  
+```javascript
+// Sets both x and y midpoints to 1 and 1, respectively. 
+FSM.setMid(thing, 1, 1);
+
+// The above is equivalent to the following 2 function calls. 
+FSM.setMidX(thing, 1);
+FSM.setMidY(thing, 1);
+```
+Going along with setting midpoints, you can do that based on the midpoint of another Thing. Using `setMidXObj`, `setMidYObj`, and `setMidObj` you can shift a Thing so that its horizontal and/or vertical midpoint(s) are centered on that of another Thing.  
+```javascript
+// Aligns a Thing's horizontal and vertical midpoints with that of anotherThing.
+// This is equivalent to the following 2 function calls combined.
+FSM.setMidObj(thing, otherThing);
+
+// Aligns just the horizontal midpoint. 
+FSM.setMidXObj(thing, otherThing);
+
+// Aligns just the vertical midpoint. 
+FSM.setMidYObj(thing, otherThing);
+```
+
+#### Slide  
+`slideToX` or `slideToY` will slide a Thing toward a target `x` or `y`, but also limits the total distance allowed (distance computed from the Thing's original midpoint).  
+```javascript
+// Slides a Thing to x position 10, moving a max distance of 5. 
+FSM.slideToX(thing, 10, 5);
+
+// Slidies a Thing to y position 10, moving a max distance of 5. 
+FSM.slideToY(thing, 10, 5);
+```
+
+## Opacity  
+Opacity defines how transparent a Thing is. A Thing's opacity is defined in the `opacity` property and ranges from 0 to 1 (0 = fully transparent, 1 = fully visible).  
```

Comment:
You say opacity thrice, which is a lot. Instead of "A Thing's opacity", say "It's".